### DISCUSSION TITLE
Daily / Nighlty Builds

### DISCUSSION THREAD

**Discussion by thismarty** [flxzt/rnote#1149] [Development]
Just curious, are there daily or nightly builds, perhaps buildbot-mediated?

**Comment by Doublonmousse** [comment]
No, there's no nightly build at the moment. If we were to do that, it would probably be one build per week or two though.

**Reply by thismarty** [reply]
Weeklies would be fine.   ;-)

**Comment by flxzt** [comment]
Is there actually the need for nightlies? I am wondering were you are coming from, are you interested in testing the latest features and looking for bugs or are you looking to daily drive nightlies because the latest features were not included in a release yet?

**Reply by thismarty** [reply]
Indeed, I want to help test.  It's really the only way I can contribute right now, but I want to help becasue RNote is pretty amazing.

**Reply by flxzt** [reply]
Alright, we should look into building maybe nightly flatpaks then!

**Comment by flxzt** [comment]
I just added a nightly workflow. Check out the [nightlies section](https://github.com/flxzt/rnote/blob/main/BUILDING.md#nightlies) for instructions how to download and install them.

**Comment by thismarty** [comment]
Aweseom news!

The 0.11 build seems to be a little more stable and the new features are great.

Thanks for this, and also thanks for starting this awesome project!  I think it could be the best drawing/sketching app available one day.


-------


I just added a nightly workflow. Check out the nightlies section<https://github.com/flxzt/rnote/blob/main/BUILDING.md#nightlies> for instructions how to download and install them.
